gui/default/syncthing/core/syncthingController.js

```diff
@@ -2096,12 +2096,19 @@ angular.module('syncthing.core')
             }
         };
 
-        $scope.otherDevices = function (devices) {
+        $scope.otherDevices = function(devices) {
             if (devices === undefined) {
                 devices = $scope.deviceList();
             }
-            return devices.filter(function (n) {
-                return n.deviceID !== $scope.myID;
+
+            var ids = devices.map(function (device) {
+                return device.deviceID
+            }).filter(function (id) {
+                return id !== $scope.myID
+            });
+
+            return $scope.deviceList().filter(function(n) {
```

Comment:
```suggestion
            return $scope.deviceList().filter(function (n) {
```
This actually isn't 100% consistent in the file, but the vast majority of the code does use a space between function and its paremeters.

Reply:
Yeah, but currently, in the controller file, there are 320+ cases of `function (n)` and only 20+ cases of `function(n)`, so there is that. It would be nice to have a script or something that would automatically format the code (or warn when the formatting doesn't match the convention). I think this is already done for the Go code but not JS, HTML, etc.

Reply:
My understanding is that named functions have the function keyword followed by the name (space separated of course) and then parens without a space. If you just leave out the name from that, the space is usually not removed though, making it more obvious that the function is not named "function".